packages/psss-server/src/routes/reports/DeleteCountryWeeklyReportRoute.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { Route } from '../Route';
+
+const SURVEY_CODE = 'PSSS_WNR';
+
+export class DeleteCountryWeeklyReportRoute extends Route {
+  async buildResponse() {
+    const { week } = this.req.query;
+    const { organisationUnitCode } = this.req.params;
+
+    const existingSurveyResponse = await this.meditrakConnection?.findSurveyResponse(
+      SURVEY_CODE,
+      organisationUnitCode,
+      week,
+    );
+
+    if (existingSurveyResponse) {
+      return this.meditrakConnection?.deleteSurveyResponse(existingSurveyResponse);
+    }
+
+    return 'No existing survey response';
```

Comment:
I just returned a string here since this didn't feel like an error case? Though I suppose you could argue it's a `404`...  I'm not too sure either way

Reply:
Yeah I'm not sure, but no error seems fine